src/plugins/gmx/gmxplugin.cpp

```diff
@@ -51,41 +51,40 @@ bool GmxPlugin::write(const Map *map, const QString &fileName)
     stream.setAutoFormatting(true);
     stream.setAutoFormattingIndent(2);
 
-    stream.writeComment("This Document is generated by GameMaker, if you edit it by hand then you do so at your own risk!");
+    stream.writeComment("This Document is generated by Tiled, if you edit it by hand then you do so at your own risk!");
 
     stream.writeStartElement("room");
-    stream.writeStartElement("caption");
-    stream.writeEndElement();
 
     stream.writeTextElement("width", QString::number(map->tileWidth() * map->width()));
-
     stream.writeTextElement("height", QString::number(map->tileHeight() * map->height()));
 
-    stream.writeTextElement("vsnap", QString::number(map->tileHeight()));
-    stream.writeTextElement("hsnap", QString::number(map->tileWidth()));
```

Comment:
Please add back these `vsnap` and `hsnap` variables, because without them GameMaker will display empty red-colored input boxes for these fields. Writing the tile size in there seems reasonable.